test/ussd_popi_user_data.test.js

```diff
@@ -689,10 +689,7 @@ describe('app', function() {
                                 ].join('\n')
                             })
                             .check(function(api) {
-                                // TODO: check that actually has been removed
```

Comment:
Maybe here we should check that we have actually deleted the `operator` and `msisdn` fields off of the contact's answers?